knowledge-base/old-webforms-viewer-does-not-display-reports-after-refresh.md

```diff
@@ -0,0 +1,47 @@
+---
+title: The Old ASP.NET Webforms ReportViewer Does Not Display Reports after Refresh
+page_title: The Old ASP.NET Webforms ReportViewer Does Not Display Reports after Refresh
+description: "Learn how to display reports after refreshing the old ASP.NET WebForms ReportViewer."
+slug: old-webforms-viewer-does-not-display-reports-after-refresh
+tags: telerik, reporting, troubleshoot, asp, net, webforms, old, reportviewer, does, not, display, reports, after, refresh
+published: True
+type: troubleshooting
+res_type: kb
+---
+
+## Environment
+
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting</td>
+		</tr>
+		<tr>
+			<td>Version</td>
+			<td>Q2 2012 and newer</td>
+		</tr>
+	        <tr>
+			<td>Report Viewers</td>
+			<td>WinForms, WPF, ASP.NET</td>
```

Comment:
Related only to the legacy AAP.NET WebForms Viewer

Include a note about this so that it is clear to the readers. Most of them don't use this viewer anymore and we recommend them to migrate to the Html5 Viewer